Updated phpstorm.cfg

cafferata opened the pull request:
Updated phpstorm.cfg to support version 2017.1 and 2017.2.

MadeOfMagicAndWires commented:
[Please add your change to the WIP Changelog](https://github.com/lra/mackup/blob/master/CONTRIBUTING.md)

cafferata commented:
Hi @MadeOfMagicAndWires! Sorry for missing out your reaction. I added my changes to the WIP `CHANGELOG.md` and updated the branch with the latests commits and the latests version of PHPStorm 2017.3.
